nvm.sh

```diff
@@ -1545,6 +1568,13 @@ nvm() {
       if [ "_$1" = "_-s" ]; then
         nobinary=1
         shift
+        if [ "_$1" = "_-j" ]; then
```

Comment:
however, this `-j` only is processed under `-s` - shouldn't it be parsed regardless, even if it's ignored when installing from a binary?